dataframe/dataframe.go

```diff
@@ -1864,3 +1864,36 @@ type Matrix interface {
 	Dims() (r, c int)
 	At(i, j int) float64
 }
+
+// Describe prints the summary statistics for each column of the dataframe
+func (df DataFrame) Describe() DataFrame {
+	labels := series.Strings([]string{
+		"mean",
+		"stddev",
+		"min",
+		"25%",
+		"50%",
+		"75%",
+		"max",
+	})
+	labels.Name = "column"
+
+	ss := []series.Series{labels}
+
+	for _, col := range df.columns {
+		newCol := series.Strings([]string{
```

Comment:
I did modify this behavior to return a Float series for numeric (int, float, bool) series and a String series for strings.